xlators/cluster/afr/src/afr-common.c

```diff
@@ -7876,3 +7876,49 @@ afr_ta_dict_contains_pending_xattr(dict_t *dict, afr_private_t *priv, int child)
 
     return _gf_false;
 }
+
```

Comment:
Lookup should also not select iatt from outcast right? Is that change not necessary? In synchronous replication this may not be necessary because other responses would have marking. But not sure about halo.